vectorizers/token_cooccurrence_vectorizer.py

```diff
@@ -940,11 +928,12 @@ class TokenCooccurrenceVectorizer(BaseEstimator, TransformerMixin):
     normalization: str ("Bayesian" or "frequentist")
         Sets the feature normalization to be the frequentist L_1 norm or the Bayesian (Dirichlet Process) normalization
 
-    coo_max_memory: str (optional, default = "2 GiB")
-        This value, giving a memory size in k, M, G or T, describes how much memory to set for acculumating the
+    coo_max_memory: str (optional, default = "0.5 GiB")
```

Comment:
I know that we accept piles of different units such as G, GB, Gi, GiB (with those being slightly different).  That's not a problem, but since we only mention the basic ones in the doc string I think we should probably have the default match one of those to reduce possible confusion.